src/controllers/accounts.js

```diff
@@ -115,6 +115,7 @@ function * putResource () {
   if (account.name) {
     request.assert.strictEqual(account.name.toLowerCase(), name.toLowerCase(),
       'Account name must match the one in the URL')
+    account.name = account.name.toLowerCase()
```

Comment:
I don't know, but tests pass so if they aren't supposed to be we should add a test case for that